lib/cloud_foundry.rb

```diff
@@ -11,6 +11,15 @@ def self.vcap_data
     end
   end
 
+  def self.app_url
+    if self.is_environment?
+      # (arbitrarily) use the shortest assigned route, if there are multiple
+      self.vcap_data['application_uris'].min_by(&:length)
+    else
+      nil
```

Comment:
I too like the explicit return from else when the `nil` is important. Ruby (like Perl) relies on implicit returns from methods and defaults to `nil` but that implicit-ness is both hard on newbies and can be non-obvious on later refactorings.